packages/mux-player/src/index.ts

```diff
@@ -656,6 +681,41 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     this.setAttribute(PlayerAttributes.THEME, `${val}`);
   }
 
+  /**
+   * Get the theme attributes in a plain object (camelCase keys).
+   * This includes already defined attributes. e.g. streamType, disabled, etc.
+   */
+  get themeProps() {
```

Comment:
It's a bit confusing that if I set something via `themeProps`, if I get it again, I don't get the same object back.

Reply:
Should we be filtering out some of the "well-known" attributes like `style` and `exportparts`? Because, I get set `theme-style` or `theme-exportparts` and it'll override the already set attribute, which seems dangerous.